pontoon/base/tests/test_views.py

```diff
@@ -140,3 +146,42 @@ def test_no_subpage_no_stats_in_current_locale(self):
         with patch('pontoon.base.views.render', wraps=render) as mock_render:
             self.client.get(url)
             assert_true('part' not in mock_render.call_args[0][2])
+
+
+class ContributorsTests(TestCase):
+    @patch('pontoon.base.models.UserTranslationsManager.with_translation_counts')
+    @patch('pontoon.base.views.render', wraps=render)
+    def test_default_period(self, mock_render, mock_translations_manager):
+        """
+        Checks how view handles default period.
+        """
+        response = self.client.get('/contributors/')
+        assert_true(mock_render.call_args[0][2]['period'] == None)
+        assert_true(mock_translations_manager.call_args[0][0] == None)
+
+    @patch('pontoon.base.models.UserTranslationsManager.with_translation_counts')
+    @patch('pontoon.base.views.render', wraps=render)
+    def test_invalid_period(self, mock_render, mock_translations_manager):
+        """
+        Checks how view handles invalid period, it result in period being None - displays all data.
+        """
+        # If period parameter is invalid value
+        response = self.client.get('/contributors/?period=invalidperiod')
+        assert_true(mock_render.call_args[0][2]['period'] is None)
+        assert_true(mock_translations_manager.call_args[0][0] == None)
```

Comment:
`assert_equal` here and the other spots you use `assert_true` for equality. Also, you're testing if the value is `None`, which means you should use `is` to test for identity.